.github/workflows/presubmit.yml

```diff
@@ -3,6 +3,9 @@ on:
   push:
   pull_request:
 
+env:
+  GRADLE_BUILD_ACTION_CACHE_DEBUG_ENABLED: true
```

Comment:
Please add a comment on what this does with a link to docs https://github.com/marketplace/actions/gradle-build-action#cache-debugging-and-analysis

Reply:
Done